### ISSUE TITLE
🐛Bug:

### ISSUE THREAD

**Issue by isaacfink** [melt-ui/melt-ui#67]
**Describe the bug**
Error in console when opening a dialog

**To Reproduce**
copy the example from the docs

**Expected behavior**
dialog should open (and close when clicking on the overlay)

**Screenshots**

**Desktop (please complete the following information):**
 macos
brave (chromium)
1.51.118

this is the error
```
dex.js:351 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at updateTabbableNodes2 (index.js:351:13)
    at Object.activate (index.js:723:9)
    at activate (focusTrap.js:12:38)
    at useFocusTrap (focusTrap.js:39:13)
    at addAction (builder.js:104:24)
u
```

**Comment by TGlide** [comment]
This has been fixed on [v0.11.1](https://github.com/melt-ui/melt-ui/releases/tag/v0.11.1)

I'm going to close this, but please let me know if you run into issues after updating!

**Comment by isaacfink** [comment]
Just upgraded and it works, thanks
